initramfs/scripts/ubuntu-core-functions

```diff
@@ -54,8 +62,6 @@ do_root_mounting()
 	fi
 
 	# mount writable rw
-	#path=$(get_partition_from_label "$writable_label")
-	# FIXME: we're handling only the encrypted data case
```

Comment:
We want this FIXME now above in the "label=ubuntu-data" case, no? I.e. with the above change it will only handle the encrypted partition now (which is not a regression but we still should have the FIXME).

Reply:
Correct. There's also an extra delay there that can now be removed. Will issue a fix soon.